pages/index.tsx

```diff
@@ -28,6 +31,18 @@ const HEARTS_IMAGE: Record<CountryVariant, ReactNode> = {
   pl: <Image src={HeartsUkraine_PL} alt="Polskie i Ukraińskie flagi w sercach" width={140} height={164} />,
 };
 
+interface Languages {
```

Comment:
Rename this `CountryVariantFlag` or something specific like that. We already have a type `Language` that is heavily used throughout the app.

Reply:
renamed